### PR TITLE
[BACKLOG-32232] Fix minor issues in File Open/Save Dialog

### PR DESCRIPTION
**Pull request by NJtwentyone**
- add common filter options for VFS dialog
- added SelectionAdapterOptions constructors to use FilterType arguments

### CODE REVIEW

**Comment by buildguy** [comment]
### :warning: Build finished in 10m 51s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
plugins/core/ui,plugins/file-open-save-new/core,ui
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 1 test(s) failed:

<details><summary><b>org.pentaho.di.ui.core.events.dialog.SelectionAdapterOptionsTest.testSelectionAdapterOptions</b> (click to expand)</summary><p>

```
expected: org.pentaho.di.ui.core.events.dialog.FilterType<CSV> but was: java.lang.String<CSV>
```

</p></details>

  
Tests run: **1121**, Failures: **1**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-7044/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 10m 53s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
plugins/core/ui,plugins/file-open-save-new/core,ui
```
---
:ok_hand: All tests passed!
 
Tests run: **1121**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-7044/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 10m 49s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
plugins/core/ui,plugins/file-open-save-new/core,ui
```
---
:ok_hand: All tests passed!
 
Tests run: **1121**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-7044/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: plugins/file-open-save-new/core/src/main/javascript/app/app.component.js [hunk 1]
```diff
@@ -458,38 +457,6 @@ define([
       }
     }
 
-    /**
```
Comment (NJtwentyone) [inline plugins/file-open-save-new/core/src/main/javascript/app/app.component.js]: code is no longer needed, from backlog-31207 the function vm.onOpenClick() replaced functionality.

**Pull request merged**